orte/mca/ess/base/ess_base_std_prolog.c

```diff
@@ -56,6 +59,10 @@ int orte_ess_base_std_prolog(void)
             goto error;
         }
     }
+    /* check for OPAL_PREFIX */
+    if (NULL != (evar = getenv("OPAL_PREFIX"))) {
+        opal_setenv("PMIX_PREFIX", evar, false, &environ);
```

Comment:
about the second case: that is fine to provide `PMIX_PREFIX`, the problem is it won't be propagated.